packages/html/src/pager/tests/pager-adaptive.tsx

```diff
@@ -1,4 +1,6 @@
-import { Pager } from '..';
+import { PagerAdaptive } from '../templates/pager-adaptive';
+import { PagerInput } from '../templates/pager-input';
+import { PagerNumeric } from '../templates/pager-numeric';
 
```

Comment:
All templates must be defined in the **index.tsx** file (like here- https://github.com/telerik/kendo-themes/blob/develop/packages/html/src/combobox/index.ts). Hence, in each test import only the components you need (like here - https://github.com/telerik/kendo-themes/blob/develop/packages/html/src/combobox/tests/combobox-adaptive.tsx#L1)